setup.py

```diff
@@ -12,74 +12,46 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from __future__ import absolute_import
-from __future__ import print_function
-
-import sys
 from glob import glob
-from os.path import basename
-from os.path import splitext
-
-from setuptools import find_packages
-from setuptools import setup
-from setuptools.command.test import test as TestCommand
-
-
-# Allows one to simply > python3 setup.py test
-class PyTest(TestCommand):
-    user_options = [('pytest-args=', 'a', "Arguments to pass to py.test")]
+from os.path import basename, splitext
 
-    def initialize_options(self):
-        TestCommand.initialize_options(self)
-        self.pytest_args = []
+from setuptools import find_packages, setup
 
-    def finalize_options(self):
-        TestCommand.finalize_options(self)
-        self.test_args = []
-        self.test_suite = True
 
-    def run_tests(self):
-        # import here, cause outside the eggs aren't loaded
-        import pytest
-        errno = pytest.main(self.pytest_args)
-        sys.exit(errno)
-
-
-with open('README.md') as f:
+with open("README.md", encoding="utf-8") as f:
     long_description = f.read()
 
+
 setup(
-    cmdclass={'test': PyTest},
-    name='vinyldns-python',
-    version='0.9.8',
-    packages=find_packages('src'),
-    package_dir={'': 'src'},
-    py_modules=[splitext(basename(path))[0] for path in glob('src/*.py')],
+    name="vinyldns-python",
+    version="0.9.7",
+    packages=find_packages("src"),
+    package_dir={"": "src"},
+    py_modules=[splitext(basename(path))[0] for path in glob("src/*.py")],
     include_package_data=True,
     zip_safe=False,
-    url='https://github.com/vinyldns/vinyldns-python',
-    license='Apache Software License 2.0',
-    author='vinyldns',
-    author_email='vinyldns-core@googlegroups.com',
+    url="https://github.com/vinyldns/vinyldns-python",
+    license="Apache Software License 2.0",
+    author="vinyldns",
+    author_email="vinyldns-core@googlegroups.com",
     classifiers=[
-        'Development Status :: 5 - Production/Stable',
-        'Intended Audience :: Developers',
-        'Topic :: Internet :: Name Service (DNS)',
-        'License :: OSI Approved :: Apache Software License',
-    ],
-    keywords=[
-        'dns', 'python', 'vinyldns',
+        "Development Status :: 5 - Production/Stable",
+        "Intended Audience :: Developers",
+        "Topic :: Internet :: Name Service (DNS)",
+        "License :: OSI Approved :: Apache Software License",
+        "Programming Language :: Python :: 3",
     ],
```

Comment:
These versions are 5+ years old and may have compatibility issues with Python 3.12. Consider updating to:
```python
tests_require=[
    "responses>=0.23.0",
    "pytest>=7.0.0",
],
```
Note: `tox.ini` already uses newer versions, so these should match.